server/migrations/20201104100526-create-task.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Tasks', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+        validate: {
+          notEmpty: {
+            args: true,
+            msg: "Title is required"
+          }
+        }
+      },
+      description: {
+        type: Sequelize.STRING
+      },
+      CategoryId: {
```

Comment:
ketika migrasi create Task gagal,
ERROR: relation "Categories" does not exist
terjadi karena create Task di migrasi duluan padahal tabel Categories belum ada, tolong diperbaiki ya

Reply:
iya kak yg ini sdh saya perbaiki kemaren, cuman belum dipush lagi